joomla/joomla-cms
# New TinyMCE: Correcting default TinyMCE params when installing Joomla

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/13344#issuecomment-269857020

### Summary of Changes
Updating default tiny params in joomla.sql
### Testing Instructions
Patch and install a new staging. Check that the Warning message is not displayed anymore when editing an article and when editing the plugin
### Documentation Changes Required
